### ISSUE TITLE
projectgenerator gets stuck at model lookup

### ISSUE THREAD

**Issue by zigertiger** [opengisch/projectgenerator#129]
The projectgenerator gets stuck at http://models.kkgeo.ch while looking up a cantonal model, eg. GL_Nutzungsplanung_V1 (see http://models.geo.gl.ch/DBU/GL_Nutzungsplanung_V1.ili).

Our model repository is accessable either via http or via https, and it is - IMHO - properly registered at models.kkgeo.ch/ilisite.xml.
At least, other tools that work with model repositories are able to locate our cantonal models...

![pgen-stuck-at-model-lookup](https://user-images.githubusercontent.com/25387992/34036751-996733c0-e186-11e7-8403-79b3677d29f5.png)

**Comment by m-kuhn** [comment]
Can you delete .ilicache and try again? Maybe also inside the plugin delete libili2db/bin

**Comment by zigertiger** [comment]
.ilicache is re-established properly, containing all repository folders and respective XML-files. Nevertheless, the projectgenerator still gets stuck at models.kkgeo.ch --- nothing changed...

**Comment by m-kuhn** [comment]
I could reproduce this locally. Executing the ili2db command from the command line gave me the same result (i.e. freeze at kkgeo), so it looks like an ili2db issue.

However, after deleting the .ilicache folder, rerunning from the command line succeeds and also rerunning from within the project generator.

Trace log:

```
 mkuhn  ~  java -jar /home/mkuhn/dev/projectgenerator/projectgenerator/libili2db/bin/ili2pg-3.10.6/ili2pg.jar --schemaimport --dbhost localhost --dbusr postgres --dbpwd postgres --dbdatabase qgis_test --dbschema nutzungsplanung --setupPgExt --coalesceCatalogueRef --createEnumTabs --createNumChecks --coalesceMultiSurface --coalesceMultiLine --strokeArcs --beautifyEnumDispName --createUnique --createGeomIdx --createFk --createFkIdx --createMetaInfo --smart2Inheritance --defaultSrsCode 2056 --models GL_Nutzungsplanung_V1 --trace
Info: dburl <jdbc:postgresql://localhost/qgis_test>
Info: dbusr <postgres>
Info: postConnect(): CREATE EXTENSION IF NOT EXISTS postgis; (PgCustomStrategy.java:61)
Info: postConnect(): CREATE EXTENSION IF NOT EXISTS "uuid-ossp"; (PgCustomStrategy.java:61)
Info: ili2pg-3.10.6-20170821
Info: ili2c-4.7.3-20170524
Info: iox-ili-1.19.6-master-20170821
Info: java.version 1.8.0_151
Info: user.name <mkuhn>
Info: maxMemory 3595776 KB
Info: dbschema <nutzungsplanung>
Info: dburl <jdbc:postgresql://localhost/qgis_test>
Info: dbusr <postgres>
Info: postConnect(): CREATE EXTENSION IF NOT EXISTS postgis; (PgCustomStrategy.java:61)
Info: postConnect(): CREATE EXTENSION IF NOT EXISTS "uuid-ossp"; (PgCustomStrategy.java:61)
Info: databaseProduct <PostgreSQL>
Info: databaseVersion <9.6.6>
Info: driverName <PostgreSQL Native Driver>
Info: driverVersion <PostgreSQL 9.4.1208.jre6>
Info: postGISVersion <POSTGIS="2.3.3 r15473" GEOS="3.6.1-CAPI-1.10.1 r0" PROJ="Rel. 4.9.3, 15 August 2016" GDAL="GDAL 2.1.4, released 2017/06/23" LIBXML="2.9.4" RASTER>
Info: compile models...
Info: getModelRepos(): pathmap key <%ILI_FROM_DB> (Main.java:753)
Info: getModelRepos(): pathmap key <%XTF_DIR> (Main.java:753)
Info: getModelRepos(): pathmap key <%JAR_DIR> (Main.java:753)
Info: setRepositories(): uri <http://models.interlis.ch/> (IliManager.java:87)
Info: setRepositories(): uri </home/mkuhn/dev/projectgenerator/projectgenerator/libili2db/bin/ili2pg-3.10.6> (IliManager.java:87)
Info: lookup model <GL_Nutzungsplanung_V1> in repository <http://models.interlis.ch/>
Info: getIliFiles(): read ilimodels.xml from <http://models.interlis.ch/>... (RepositoryAccess.java:108)
Info: lookup model <GL_Nutzungsplanung_V1> in repository </home/mkuhn/dev/projectgenerator/projectgenerator/libili2db/bin/ili2pg-3.10.6>
Info: getIliFiles(): scan ili-files in folder </home/mkuhn/dev/projectgenerator/projectgenerator/libili2db/bin/ili2pg-3.10.6>... (RepositoryAccess.java:94)
Info: getIliSite(): read ilisite.xml from <http://models.interlis.ch/>... (RepositoryAccess.java:129)
Info: getIliSite(): read ilisite.xml from </home/mkuhn/dev/projectgenerator/projectgenerator/libili2db/bin/ili2pg-3.10.6>... (RepositoryAccess.java:129)
Info: lookup model <GL_Nutzungsplanung_V1> in repository <http://models.geo.admin.ch>
Info: getIliFiles(): read ilimodels.xml from <http://models.geo.admin.ch>... (RepositoryAccess.java:108)
Info: getIliSite(): read ilisite.xml from <http://models.geo.admin.ch>... (RepositoryAccess.java:129)
Info: lookup model <GL_Nutzungsplanung_V1> in repository <http://models.kkgeo.ch>
Info: getIliFiles(): read ilimodels.xml from <http://models.kkgeo.ch>... (RepositoryAccess.java:108)
```

**Comment by pramisters** [comment]
I have the same problem (other model), but only if I choose the model Name ("Models") as Input. If I choose the ili file ("Interlis Datei") it works fine.
